datamodel/brk/brk2.0_oracle_views.sql

```diff
@@ -462,6 +462,27 @@ COMMENT ON MATERIALIZED VIEW mb_percelenkaart IS
     * lon: coordinaat als WSG84,
     * begrenzing_perceel: perceelvlak';
 
+-- BRMO-336: toevoegen van zakelijke rechten die het eigendomsrecht belasten
+CREATE OR REPLACE VIEW vb_util_zk_recht_op_koz 
+            (
+             identificatie,
+             rustop_zak_recht
+            )
+AS 
+SELECT  qry.identificatie,
+        qry.rustop_zak_recht
+FROM    ( 
+            SELECT ribm.isbelastmet                                     AS identificatie,
+                   zakrecht.rustop                                      AS rustop_zak_recht
+            FROM recht_isbelastmet ribm
+
+            LEFT JOIN recht zakrecht ON ribm.zakelijkrecht = zakrecht.identificatie
+
+            UNION ALL
+            SELECT zakrecht.identificatie,
+                   zakrecht.rustop                                      AS rustop_zak_recht
+            FROM recht zakrecht) qry
+WHERE SUBSTR(qry.identificatie, 1, INSTR(qry.identificatie, ':') - 1) = 'NL.IMKAD.ZakelijkRecht';
```

Comment:
Oracle XE Online geeft de volgende foutmelding na deze regels: 

```
SP2-0734: unknown command beginning "LEFT JOIN ..." - rest of line ignored.
SP2-0042: unknown command "UNION ALL" - rest of line ignored.
	    FROM recht zakrecht) qry
                               *
ERROR at line 3:
ORA-00933: SQL command not properly ended
```
@matthijsln zou je mee kunnen kijken waar het aan ligt? Mogelijk is er toch een syntaxerror die ik over het hoofd zie.

Reply:
Het lijkt erop alsof je SQL*Plus gebruikt, dat werkt niet goed met dit script vanwege [lege regels](https://docs.oracle.com/cd/E11882_01/server.112/e16604/ch_four.htm#i1039663).

Reply:
sqlplus wordt gebruikt in de CI en in de installatie handleiding, dus dat moet werken

Reply:
Ah dat wist ik niet, dan moet denk de lege regel weggehaald worden.